src/api/eth.rs

```diff
@@ -366,14 +422,16 @@ mod tests {
     Eth:block:block_by_hash, BlockId::Hash(0x123.into()), true
     =>
     "eth_getBlockByHash", vec![r#""0x0000000000000000000000000000000000000000000000000000000000000123""#, r#"true"#];
-    Value::Null => ()
+    ::serde_json::from_str(EXAMPLE_BLOCK).unwrap()
+    => ::serde_json::from_str::<Block>(EXAMPLE_BLOCK).unwrap()
```

Comment:
Right, for simple structures it indeed might not make much sense, but still choosing right params to deserialize from is important and it's part of the logic, and some methods will do different things depending on the output.

Maybe instead of deserializing structs from strings we should just put `Default::default()`?